internal/addrs/removable.go

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) HashiCorp, Inc.
```

Comment:
why are we adding new files with the hashicorp copyright?

Reply:
I waited for the [final copywrite notice](https://github.com/opentofu/opentofu/issues/454#issuecomment-1904213569).
Updated all the newly added files with the OpenTofu and HashiCorp headers, as [discussed on Slack](https://opentofucommunity.slack.com/archives/C05R0FD0VRU/p1706012969801869?thread_ts=1705997490.521719&cid=C05R0FD0VRU).